scripts/mass-invite-users/massInviteUsers.js

```diff
@@ -0,0 +1,116 @@
+const AUTH_TOKEN = "";
+const TRACKER_GRAPHQL_URI = "";
+
+if (!AUTH_TOKEN) {
+  console.error("You must enter your auth token!");
+}
+
+if (!TRACKER_GRAPHQL_URI) {
+  console.error("You must enter the Tracker graphql URI!");
+}
+
+const csv2json = (str, delimiter = ",") => {
+  const titles = str
+    .slice(0, str.indexOf("\n"))
+    .replace(/["']/g, "")
+    .split(delimiter)
+    .map((str) => str.trim());
+  const rows = str.slice(str.indexOf("\n") + 1).split("\n");
+  return rows.map((row) => {
+    const values = row.split(delimiter);
+    return titles.reduce((object, curr, i) => {
+      object[curr] = values[i].replace(/["']/g, "").trim();
+      return object;
+    }, {});
+  });
+};
+
+const getOrg = async (orgSlug) => {
+  const res = await fetch(TRACKER_GRAPHQL_URI, {
+    body: JSON.stringify({
+      query: `query {
+            findOrganizationBySlug(orgSlug: "${orgSlug}") {
+                id
+                name
+                slug
+            }
+        }`,
+    }),
+    headers: {
+      Accept: "application/json",
+      Authorization: AUTH_TOKEN,
+      "Content-Type": "application/json",
+    },
+    method: "POST",
+  });
+  return (await res.json()).data;
+};
+
+const inviteUser = async ({ email, role, orgId }) => {
+  const res = await fetch(TRACKER_GRAPHQL_URI, {
+    body: JSON.stringify({
+      query: `mutation {
+            inviteUserToOrg(input: {
+                userName: "${email}",
+                requestedRole: ${role},
+                orgId: "${orgId}",
+                preferredLang: ENGLISH
```

Comment:
This parameter is used to decide what language the invite email is sent. For users that already exist it chooses their preferred language, but for users who don't have an account, we should have a bilingual message sent.

Reply:
Bilingual email being sent now :+1: